fmf/utils.py

```diff
@@ -912,34 +913,22 @@ def dict_to_yaml(data, width=None, sort=False):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 class JsonSchemaValidationResult(NamedTuple):
-    """ Represents JSON Schema validation result """
-
+    """Represents JSON Schema validation result."""
```

Comment:
Can we keep these as they are until there's a concensus on the formatting?

Reply:
reverted in fixup.